sdk/openai/openai/src/index.ts

```diff
@@ -2,18 +2,55 @@
 // Licensed under the MIT license.
 
 /**
- * [Azure OpenAI](https://learn.microsoft.com/azure/cognitive-services/openai/overview)
- * provides REST API access to OpenAI's powerful language models including the GPT-3,
- * Codex and Embeddings model series. In addition, the new GPT-4 and ChatGPT (gpt-35-turbo)
- * model series have now reached general availability. These models can be easily adapted
- * to your specific task including but not limited to content generation, summarization,
- * semantic search, and natural language to code translation.
+ * THIS IS AN AUTO-GENERATED FILE - DO NOT EDIT!
  *
- * @packageDocumentation
```

Comment:
can we add back this comment manually until it gets supported in te custom tool?